boards/nxp/frdm_ke17z/frdm_ke17z_defconfig

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+
+# GPIO Controller
+CONFIG_GPIO=y
+
+# Clock Control
+CONFIG_CLOCK_CONTROL=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=72000000
```

Comment:
Is this accurate? The documentation states the board is clocked at 48MHz, and this value should be equal to the clock frequency the SYSTICK is running at on boards using it (which this M0+ will by default). A good way to check if this value is correct is to use a sample like `samples/basic/blinky`, and make sure the LED toggles once per second using an oscilloscope or signal analyzer

Reply:
Thanks for reminding, the actual frequency should be 48000000, which should be consistent with the index.rsy document.